README.md

```diff
@@ -177,6 +177,21 @@ $ dotenv-linter --skip UnorderedKey EndingBlankLine
 .env:2 DuplicatedKey: The FOO key is duplicated
 ```
 
+If you need to view all available checks, you can use the argument `--show-checks` (will be available in [v2.0.0](https://github.com/dotenv-linter/dotenv-linter/issues/199)):
+
+```shell script
+$ dotenv-linter --show-checks
```

Comment:
I wouldn't write that list out here, it gets outdated really fast if we would add new checks or remove something and to hold them in sync is easily forgotten.

What I would rather do is to write the checkname after each headline of the check i.e.

```
### Duplicated Key (`DuplicatedKey`)
```

or something.
What do you think @mgrachev ?

Reply:
I think this is just an example and I need to understand that the result may vary from version to version.
That's why nowhere in the code did I start adding tests with a strict list, but README is just an example.

That is why I consider it important to add an `help` argument, as I described in [Ideas issue](https://github.com/dotenv-linter/dotenv-linter/issues/23#issuecomment-633460192)